Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -151,10 +164,34 @@ private void UpdateRaycast()
             {
                 if (oldFocusedObject != null)
                 {
+                    IInteractable oldInteractable = oldFocusedObject.GetComponent<IInteractable>();
```

Comment:
I have concerns here doing a GetComponent every Update frame for 2 game objects. Can we make some measurements perhaps? With and without?